app/uk/gov/hmrc/voapropertylinking/controllers/PropertyLinkingController.scala

```diff
@@ -153,11 +153,12 @@ class PropertyLinkingController @Inject()(
 
     }
 
-  def getMyClients(clientsParameters: Option[GetClientsParameters]): Action[AnyContent] = authenticated.async {
-    implicit request =>
-      propertyLinkService
-        .getMyClients(clientsParameters.getOrElse(GetClientsParameters()), None)
-        .map(clients => Ok(Json.toJson(clients)))
+  def getMyClients(
+        clientsParameters: Option[GetClientsParameters],
+        params: Option[PaginationParams]): Action[AnyContent] = authenticated.async { implicit request =>
```

Comment:
`pagination` or `paginationParams` would be better than just `params`, it's not clear what params is when you use it: `getMyClients(clientsParameters.getOrElse(GetClientsParameters()), params)`